scripts/requirements-extras.txt

```diff
@@ -19,7 +19,9 @@ Pillow
 imgtool>=1.7.1
 
 # used by nanopb module to generate sources from .proto files
-protobuf
+# Google introduced breaking changes in protobuf v4.21.0, use v3 for now
```

Comment:
The commit message says v4.20.0, but this says v4.21.0. What's going on?

Reply:
My bad.